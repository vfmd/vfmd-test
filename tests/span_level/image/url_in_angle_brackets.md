 * ![link](<url>)
 * ![link](<url(>)
 * ![link](<url)>)
 * ![link](<url)> "title")
 * ![ref id1][]
 * ![ref id1]
 * ![ref id2][]
 * ![ref id2]

[ref id1]: <url>
[ref id2]: <url)> "title"
