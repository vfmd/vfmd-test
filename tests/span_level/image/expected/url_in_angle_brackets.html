
<ul>
  <li><img src="url" alt="link" /></li>
  <li><img src="url(" alt="link" /></li>
  <li><img src="url)" alt="link" /></li>
  <li><img src="url)" alt="link" title="title" /></li>
  <li><img src="url" alt="ref id1" /></li>
  <li><img src="url" alt="ref id1" /></li>
  <li><img src="url)" alt="ref id2" title="title" /></li>
  <li><img src="url)" alt="ref id2" title="title" /></li>
</ul>
